src/main/java/stringCalculator/Operator.java

```diff
@@ -0,0 +1,30 @@
+package stringCalculator;
```

Comment:
```suggestion
package stringcalculator;
```

자바에서 패키지는 소문자로만 작성해주는 것이 관례입니다 :)